app/src/main/res/drawable/splash_page.xml

```diff
@@ -3,12 +3,12 @@
     android:height="640dp"
     android:viewportWidth="360"
     android:viewportHeight="640">
-  <group>
-    <clip-path android:pathData="M0,0h360v640h-360z M 0,0"/>
-    <path
-        android:pathData="M0,0h360v640h-360z"
-        android:fillColor="#019489"/>
-  </group>
+<!--  <group>-->
```

Comment:
if these are no longer required maybe you can remove them instead of commenting??
@rt4914

Reply:
got it , will do it

Reply:
Ack. We should not keep unnecessary comments in files.